### DISCUSSION TITLE
Crossvalidation

### DISCUSSION THREAD

**Discussion by jimmy5589** [ourownstory/neural_prophet#307] [Q&A - get help using NeuralProphet] [answered]
Hi,

Trying to use NeuralProphet with crossvalidation. There is a function crossvalidation_split_df to create the k folds provided in the package.
Could you please provide me an example how to automate this with NeuralProphet? Is there any easy way of doing it?

Thank you.
J

**Comment by ourownstory** [comment, marked as answer]
Have you seen this? 
https://github.com/ourownstory/neuralprophet-docs/blob/main/tutorials/feature-use/crossvalidation.ipynb
